charts/netdata/templates/configmap.yaml

```
@@ -37,6 +37,26 @@ data:
 
 ---
 
+{{- if .Values.k8sState.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: netdata-conf-k8s-state
+  labels:
+    app: {{ template "netdata.name" . }}
+    chart: {{ template "netdata.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  {{- range $name, $config := .Values.k8sState.configs }}
+  {{- if $config.enabled }}
+  {{ $name }}: {{ tpl $config.data $ | toYaml | indent 4 }}
```

```suggestion
  {{ $name }}: {{ tpl $config.data $ | toYaml | indent 4 | trim }}
```

this will give you:
```
data:
  go.d: |
      default_run: no
      modules:
        k8s_state: yes
```

instead of:
```
data:
  go.d:     |
      default_run: no
      modules:
        k8s_state: yes
```